locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -635,10 +645,12 @@ protected JcloudsSshMachineLocation obtainOnce(ConfigBag setup) throws NoMachine
                 sshHostAndPortOverride = Optional.absent();
             }
 
-            if (waitForSshable && skipJcloudsSshing) {
+            if (waitForSshable && skipJcloudsSshing && !windows) {
                 // once that host:port is definitely reachable, we can create the user
                 waitForReachable(computeService, node, sshHostAndPortOverride, node.getCredentials(), setup);
                 userCredentials = createUser(computeService, node, sshHostAndPortOverride, setup);
+            } else if (windows) {
```

Comment:
We want to be able to defer/never waitForWinRmAvailable. That was intent of `waitForSshable` - to turn that off. 

Historically, reason for that was to let the caller set up port forwarding. However, that is now covered in code above (but not for windows case!). So we generally don't make use of the `waitForSshable` much these days. But given it's there...

I wonder if we should generalise it to a "waitForReachable" or some such.

Don't worry about it for this PR.